CHANGELOG.asciidoc

```diff
@@ -3,6 +3,153 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.17.0]]
+=== Beats version 7.17.0
+https://github.com/elastic/beats/compare/v7.16.3...v7.17.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Add job.name in pods controlled by Jobs {pull}28954[28954]
+- Enrich kubernetes metadata with node annotations. {pull}29605[29605]
```

Comment:
@tetianakravchenko is this a breaking change?

Reply:
@andresrc sorry, my mistake, it actually should be under a `Bugfixes`